.github/workflows/lambda-upload.yaml

```diff
@@ -0,0 +1,19 @@
+name: Lambda Upload
+on:
+  release:
+    types: [published]
+jobs:
+  lambda-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: npm install
+        run: npm ci
+      - name: deploy
+        run: |
+          npx ncc build functions/activate-batch-auto-background.ts
```

Comment:
Can we add this instead into `package.json` as a `npm run build` command, and then just do `npm run build` here